sklearn/impute/_iterative.py

```diff
@@ -141,6 +141,10 @@ class IterativeImputer(_BaseImputer):
         the missing indicator even if there are missing values at
         transform/test time.
 
+    keep_missing_features : boolean, default=False
```

Comment:
This name is pretty strange?

Reply:
I think it's too long, but I couldn't find something better

Reply:
Maybe `keep_empty_features`?

Reply:
I consider `missing` a more consistent terminology, but it is not a strong opinion

Reply:
I see "missing" as being the property of a cell, but how we call a column with all "missing" cells/values? I think naming it again "missing" might be confusing. But also no strong preference here.

Reply:
Now, the attribute `indicator_` already addresses this